scripts/start.sh

```diff
@@ -45,9 +45,16 @@ set -- "${remaining_params[@]}"
 
 echo Checking whether GAE is installed in $GOOGLE_APP_ENGINE_HOME
 if [ ! -f "$GOOGLE_APP_ENGINE_HOME/appcfg.py" ]; then
-  echo "Installing Google App Engine (this may take a little while)..."
+  echo "Downloading Google App Engine (this may take a little while)..."
   mkdir -p $GOOGLE_APP_ENGINE_HOME
   curl --silent https://storage.googleapis.com/appengine-sdks/deprecated/1919/google_appengine_1.9.19.zip -o gae-download.zip
+  # $? contains the (exit) status code of previous command, which is curl here. So, if curl was successful, $? will be 0 else non-zero
```

Comment:
Is 80 char line limit still followed? I thought with widescreen monitors and all, it has become outdated. Anyways since the entire code follows the aesthetic, I'll consider doing that.